src/bitops.c

```diff
@@ -1225,13 +1227,13 @@ void bitfieldGeneric(client *c, int flags) {
                 src = getObjectReadOnlyString(o,&strlen,llbuf);
 
             /* For GET we use a trick: before executing the operation
-             * copy up to 9 bytes to a local buffer, so that we can easily
+             * copy up to 9 bytes[LOCAL_STR_OBJ_SIZE] to a local buffer, so that we can easily
```

Comment:
considering the proximity of all these lines and the big comment, i don't think this define helps readability.
maybe using `sizeof(buf)` would be a bit better instead of the define.

Reply:
Replaced the macro with sizeof now, and removed macros as you mentioned this may not improve and we have brief explanation.

Reply:
it looks better now. but still considering it's all local to one function that has a big comment explicitly referring to the value of `9`, i don't see any reason to touch that code. i don't think it has a redability issue.